packages/ui/package.json

```diff
@@ -20,13 +20,16 @@
   },
   "dependencies": {
     "@heroicons/react": "2.0.18",
+    "@radix-ui/react-dropdown-menu": "2.0.5",
+    "@radix-ui/react-form": "0.0.3",
+    "@radix-ui/react-tooltip": "1.0.6",
     "@segment/analytics-next": "1.53.0",
     "clsx": "1.2.1",
     "deepmerge": "4.3.1",
-    "encoding": "0.1.13",
-    "feed": "4.2.2",
```

Comment:
Wasn't feed used to provide a RSS feed for the blogs?

Reply:
These were fragments from the old website (no longer needed).

Reply:
Thanks